instruction/programming-exam/programming-exam.md

```diff
@@ -0,0 +1,164 @@
+# Programming Exam
+
+🖥️ [Lecture Video](#videos)
+
+This programming exam is a re-creation of your [chess phase-0](https://github.com/softwareconstruction240/softwareconstruction/blob/main/chess/0-chess-moves/chess-moves.md) assignment, but under time constraints and without TA or other support. No notes or resources are allowed except those described here.
```

Comment:
"No notes or other resources ..." Sets up the expectation that there are some type of "notes" that are allowed. I think we are getting rid of that possibility.